pdal/util/Utils.cpp

```diff
@@ -538,6 +538,9 @@ int Utils::screenWidth()
     if (ioctl(0, TIOCGWINSZ, &ws))
         return 80;
 
+    if (!ws.ws_col)
```

Comment:
We should check the return of `ioctl` and report an error if something is up.

Reply:
I meant it would be nice to tell the user what is going on. That it would fail is weird and quite unexpected. Silently defaulting to 80 seems presumptuous. 

https://man7.org/linux/man-pages/man2/ioctl.2.html

```
       EBADF  fd is not a valid file descriptor.

       EFAULT argp references an inaccessible memory area.

       EINVAL request or argp is not valid.

       ENOTTY fd is not associated with a character special device.

       ENOTTY The specified request does not apply to the kind of object
              that the file descriptor fd references.
```